packages/KDropdownMenu/KDropdownItem.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <li
+    :class="{
+      'has-divider': type !== 'link' && hasDivider,
+      'disabled': type === 'default' && disabled,
+      'danger': isDangerous,
+      'k-dropdown-selected-option': selected
+    }"
+    class="k-dropdown-item w-100"
+  >
+    <router-link
+      v-if="type === 'link' && to"
+      :data-testid="label"
+      :class="{ 'disabled': disabled, 'has-divider': hasDivider }"
+      :to="!disabled ? to : $router.currentRoute.path"
+      class="k-dropdown-item-trigger"
+    >
+      <slot>{{ label }}</slot>
+    </router-link>
+    <KButton
+      v-else-if="type === 'button'"
+      :disabled="disabled"
+      :class="classList"
+      class="k-dropdown-item-trigger"
+      v-on="listeners"
+    >
+      <slot>{{ label }}</slot>
+    </KButton>
+    <div
+      v-else
+      class="k-dropdown-item-trigger"
+    >
+      <slot>{{ label }}</slot>
+    </div>
+  </li>
+</template>
+
+<script>
+import KButton from '@kongponents/kbutton/KButton.vue'
+
+export default {
+  name: 'KDropdownItem',
+  components: { KButton },
+  props: {
+    item: {
+      type: Object,
+      default: null,
+      // Items must have a label
+      validator: (item) => item.hasOwnProperty('label')
+    },
+    /**
+     * Use this prop to add a divider above the item
+     */
+    hasDivider: {
+      type: Boolean,
+      default: false
+    },
+    isDangerous: {
```

Comment:
suggestion: what about `destructive` instead? This falls in line with "destructive" actions, and by removing the "is" prefix it's more similar to the other props. 
```suggestion
    destructive: {
```